src/redux/explorer.ts

```diff
@@ -637,20 +542,13 @@ export const explorerInit = () => async (
   // if there is another socket unsubscribe first
   if (addressSocket || assetsSocket) {
     dispatch(explorerUnsubscribe());
-    dispatch(disableFallbackIfNeeded());
   }
 
-  // Fallback to the testnet data provider
-  // if we're not on mainnnet
   const provider = await getProviderForNetwork(network);
   const providerUrl = provider?.connection?.url;
   checkForTheMerge(provider, network);
-  if (
-    isHardHat(providerUrl) ||
-    network !== Network.mainnet ||
-    forceFallbackProvider
-  ) {
-    return dispatch(fallbackExplorerInit());
+  if (isHardHat(providerUrl) || network !== Network.mainnet) {
+    return;
```

Comment:
Should we alert ourselves if this happens or is this an impossible state?

Reply:
I have a follow up note for testnet support (now we only support goerli and it isn't a big priority). I'm checking in on the hardhat branch for tests now.